applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -1070,6 +1099,28 @@ static void hogp_ready(struct bt_hogp *hids_c)
 	if (per_pm == BT_HIDS_PM_BOOT) {
 		set_peripheral_protocol_mode(per, per_pm);
 	}
+
+	/* Send collected output events dedicated to this peripheral. */
+	struct subscriber *sub = get_subscriber(per);
+
+	for (size_t orep_idx = 0; orep_idx < ARRAY_SIZE(sub->out_reports); orep_idx++) {
+		if (sub->saved_out_reports_bm & BIT(orep_idx)) {
+			/* The size of uint8_t is added here because of report id,
+			 * that is stored at the beginning of report data.
+			 */
+			size_t report_full_size = get_output_report_size(
```

Comment:
C is a spartan language - try to use shorter names. I expect `size` will be perfectly understandable local name

Also think what to do to avoid adding comments and make the code document itself. Remember that all this comments need to be maintained and there is no automated check to verify if the comment is matching the code.

Don't repeat things - it makes maintenance harder.

```
size_t size = get_output_report_size(sub->out_reports[orep_idx].report_id);